README.md

```diff
@@ -7,6 +7,10 @@ This repository contains a collection of modules for the [Safe Smart Account](ht
 - [4337 Module](./modules/4337)
 - [Allowance Module](./modules/allowances)
 
+## Examples
+
+- [Safe + 4337 + Passkeys](./examples/safe-4337-passkeys)
```

Comment:
Unrelated to this PR, but I feel like the gas metering project should also be moved here. Maybe there is a more general directory name to “examples” that would fit both workspace packages...